### DISCUSSION TITLE
Is it possible to list every third-party library used in an iOS IPA?

### DISCUSSION THREAD

**Discussion by BergHeisen** [blacktop/ipsw#473] [General]
Since the tool can sniff out functions from third-party libraries, would this be possible too?

**Comment by blacktop** [comment]
`ipsw macho info App.app --json` will tell you all the libraries loaded and then you should be able to match all non std lib frameworks used

**Reply by BergHeisen** [reply]
Hmm, it only lists dynamic librarires, does the IPA/binary contained in the ipa not have some metadata about third-party libraries that are embedded into the App? I know that static libraries are mostly stripped of all of their information, but the tool successfully lists all of the classes/protocols with `ipsw swift-dump`
